scripts/install-helpers/baremetal-coco/install.sh

```diff
@@ -8,6 +8,16 @@ KERNEL_CONFIG_MC_FILE="./96-kata-kernel-config-mc.yaml"
 SKIP_NFD="${SKIP_NFD:-false}"
 TRUSTEE_URL="${TRUSTEE_URL:-"http://kbs-service.trustee-operator-system:8080"}"
 CMD_TIMEOUT="${CMD_TIMEOUT:-900}"
+CLUSTER_HTTPS_PROXY="$(oc get proxy/cluster -o jsonpath={.spec.httpsProxy})"
```

Comment:
@fidencio we are still relying on `oc` and assignment should be after checking of `oc` command.
Also since we don't expect the admin to set this, I think better to have this assignment closer to the usage ie in `set_kernel_params_for_kata_agent`..

Reply:
Fixed, please, check the latest version.